cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1712,3 +1712,33 @@ roleRef:
   name: rbd-external-provisioner-runner
   apiGroup: rbac.authorization.k8s.io
 # OLM: END CSI RBD CLUSTER ROLEBINDING
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
This RBAC only needs to be created when the admission controller is configured. How about adding it to `webhook-config.yaml` instead of `common.yaml`?

Reply:
Added to webhook-config.yaml